content/pt-br/docs/tasks/administer-cluster/coredns.md

```diff
@@ -0,0 +1,76 @@
+---
+reviewers:
+- johnbelamaric
+title: Usando o CoreDNS para Descoberta de Serviços
+min-kubernetes-server-version: v1.9
+content_type: task
+weight: 380
+---
+
+<!-- overview -->
+Essa página descreve o processo de upgrade de CoreDNS e como instalar o CoreDNS ao invés de kube-dns.
+
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+
+<!-- steps -->
+
+## Sobre o CoreDNS
+
+[CoreDNS](https://coredns.io) é um servidor DNS flexivel e extesinvo
```

Comment:
typo: flexivel and extesinvo

Maybe it will be better "é um servidor DNS flexível e extensível".